.travis.yml

```diff
@@ -109,6 +109,18 @@ script:
       ./trillian/integration/integration_test.sh
       cd $HOME/gopath/src/github.com/google/trillian
       HAMMER_OPTS="--operations=50" ./integration/maphammer.sh 3
+  - |
```

Comment:
Do we have similar tests for AWS and GCP?

Reply:
We have something similar for GCP, as a consequence of Trillian's CI environment being on GCP. We don't have any tests for AWS though.